SelfOrganisingList.tpp

```diff
@@ -8,20 +8,77 @@ SelfOrganisingList<T>::SelfOrganisingList(int _size) : size(_size), count(0){
 
 template<typename T>
 void SelfOrganisingList<T>::add(T value) {
-    if (count < size-1){
-        auto *tmp = new Node<T>(value);
-        tab[count++] = *tmp;
+    if (count >= size) {
+
+        return; //PELNA LISTA
```

Comment:
to do: define and execute resize method, which we will call when we exceed our base size limit (there is no need for now to do an implementation for this method, we will do that later)